### PR TITLE
#176 Keras CNN Model added

### PR DESCRIPTION
**Pull request by shreya024**
I have added a Keras CNN Model with 94.9% accuracy

### CODE REVIEW

File: Model/bc_prediction_cnn.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+from tensorflow.keras.layers import Flatten, Dense, Dropout, MaxPooling1D,BatchNormalization
+from tensorflow.keras.layers import Conv1D
+from tensorflow.keras import layers,models
+
+from tensorflow.keras.optimizers import Adam
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+
+#Importing the dataset
+dataset=pd.read_csv('../Dataset/Cleaned_data_Breast_Cancer.csv')
```
Comment (srimanikantapalakollu) [inline Model/bc_prediction_cnn.py:14]: @shreya024 Try to customize this model which can able to read the data from the Flask Application and predict the result using your Model.

File: Model/bc_prediction_cnn.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+model.compile(optimizer=Adam(learning_rate=0.00005), loss = 'binary_crossentropy', metrics=['accuracy'])
+
+history = model.fit(X_train, y_train, epochs=epochs, validation_data=(X_test, y_test), verbose=1,)
+
+#testing over training dataset to create a confusuion matrix
+prediction = model.predict(X_train)
+#using 0.5 as a threshold value
+prediction_array = (prediction>0.5)
+
+results = model.evaluate(X_test, y_test)
```
Comment (srimanikantapalakollu) [inline Model/bc_prediction_cnn.py:65]: What are you doing with these results? Please make sure to write proper comments and do the appropriate changes as well.

**Review by srimanikantapalakollu — CHANGES_REQUESTED** [review]
@shreya024 Please do the requested Changes and Commit and I'll review the Same. Thanks for your Contribution. Your Code is good work but needs some improvements according to the application flow.

File: Model/bc_prediction_cnn.py (hunk 1 above)
Comment (shreya024) [reply Model/bc_prediction_cnn.py:14]: Hey @srimani-programmer I actually do not know Flask...so if you could provide a code example/resource...I would love to learn and then make the requested changes.

File: Model/bc_prediction_cnn.py (hunk 2 above)
Comment (shreya024) [reply Model/bc_prediction_cnn.py:65]: Okay sure

**Comment by shreya024** [comment]
@srimani-programmer I have deployed the CNN model using Flask. Kindly check now

**Comment by shreya024** [comment]
> @srimani-programmer @freny24  I have deployed the CNN model using Flask. Kindly check now

**Comment by srimanikantapalakollu** [comment]
@freny24 Could you tag the appropriate level label to this PR and merge it as I reviewed the code everything seems perfect.

**Comment by shreya024** [comment]
> @freny24 Could you tag the appropriate level label to this PR and merge it as I reviewed the code everything seems perfect.

Thank you for reviewing. Appreciate if you could make it a level 3 as I have made the CNN model and customized it as well for the Flask Application
